support-frontend/test/selenium/contributions/OneOffContributionsSpec.scala

```diff
@@ -59,50 +59,19 @@ class OneOffContributionsSpec extends FeatureSpec with GivenWhenThen with Before
       When("they press the Stripe payment button")
       landingPage.selectStripePayment()
 
+      And("enter card details")
```

Comment:
Previously this test got through to the /thankyou page by skipping the stripe checkout stage and sending a test token to the backend.
Now it actually populates the card details fields

Reply:
👍